mods/farming/api.lua

```diff
@@ -83,10 +83,30 @@ farming.register_hoe = function(name, def)
 		end
 	})
 	-- Register its recipe
-	minetest.register_craft({
-		output = name:gsub(":", "", 1),
```

Comment:
Okay I did what you said and it seems to work fine:

```output = name:sub(2),```

So ```gsub``` replaces parts of strings but ```sub``` searches for parts of strings, right?